### ISSUE TITLE
Classification parameters accessible from the gui

### ISSUE THREAD

**Issue by alecrimi** [pelkmanslab/TissueMAPS-OLD#1]
Some parameters for doing classification are hardcoded into the code and not accessible to users. The GUI should be expanded to allow the user to set manually those.

**Comment by hackermd** [comment]
I have optimised the grid search for the crossvalidation in `tmlib.tools.base.Classifier.classifiy_supervised()`. This improved classification accuracy (based on visual inspection). Here is a result for "randomforest" classifier:

<img width="1440" alt="screen shot 2017-01-25 at 18 49 00" src="https://cloud.githubusercontent.com/assets/6855297/22542672/43932e60-e92d-11e6-9785-487df1e90cde.png">

For the "svm" classifier, I used values for penalty parameters as recommended by the paper ["A practical guide to support vector classification"](http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.224.4115). This looks good as well.

**Comment by hackermd** [comment]
I wouldn't expose the parameters in the user interface. They are not very intuitive and a cross-validation of a wide range of parameter values should do the trick.
For an extensive cross-validation, it's important to have enough data points. So generally speaking, the more labels the better.

Tools can, in addition to a labeled layer, return a `tmlib.models.result.Plot`. We may want classifiers to return users additional graphical information about the classification result, such as a [ROC curve](https://en.wikipedia.org/wiki/Receiver_operating_characteristic).

**Comment by alecrimi** [comment]
I disagree, if a user is using a tool, (s)he is supposed to know what he is doing and (s)he should also be allowed to tune for the experiments. Let' s see if the results are satisfactory for a wide range of users, we leave it for now with the grid search and we don't add parameters.
